src/hostapi/wasapi/pa_win_wasapi.c

```diff
@@ -601,6 +601,9 @@ typedef struct PaWasapiStream
     // must be volatile to avoid race condition on user query while
     // thread is being started
     volatile BOOL running;
+    BOOL stopped;
+
+    // stream has not or is no longer started
```

Comment:
We normally put comments before the code.

Reply:
Ah, yes, that is a mistake! I'll fix that.